### ISSUE TITLE
QR code not popping up with WalletConnect

### ISSUE THREAD

**Issue by RedPlumpTomato** [pancakeswap/pancake-swap-interface#308]
**Bug Description**

QR Code does not pop up when trying to use WalletConnect. Instead, it just logs me in with a cached account. The link between that cached account and my Trust wallet on IOS therefore doesn't work.

**Steps to Reproduce**

1.) Click "Connect"
2.) Click "WalletConnect"
3.) Logs me in right away with cached account. QR code is not presented. 
   ...

**Expected Behavior**
QR code to popup

**Additional Context**
This issue occurs in Brave and Chrome.

**Comment by memoyil** [comment]
Have you disconnected your session from logged account using the app that you scanned your qr code?

**Comment by RedPlumpTomato** [comment]
Yes.  Thanks for the quick reply.

________________________________

**Comment by SamTmoud** [comment]
I've also been struggling with this exact issue. There's no cache and it happens across different browsers. Did you ever figure out what's causing it/how to fix it? I've missed out on some great ape opportunities because of this :'(

**Comment by RedPlumpTomato** [comment]
> I've also been struggling with this exact issue. There's no cache and it happens across different browsers. Did you ever figure out what's causing it/how to fix it? I've missed out on some great ape opportunities because of this :'(

Not really, I gave up and now use Metamask integration which works very well. Trust on my phone, Metamask on browser. works well.
